plugins/async-node/components/src/dsl/async.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import type { ObjectNode } from "react-json-reconciler";
+
+export type AsyncNodeProps = {
+  id?: string;
+};
+
+export const AsyncNode = React.forwardRef<ObjectNode, AsyncNodeProps>(
```

Comment:
Does this need to be a `forwardRef`? For our other DSL components they are just plain React Components.

Reply:
Well , I have taken the reference of Asset component as below :
![image](https://github.com/player-ui/player/assets/171299777/985f3e21-7530-41c2-9d06-2c9e084c01f1)

But I agree to the fact that this could be acheived using simple react components but I see the specific usage of. ref comp as  React.forwardRef is a technique used within functional components to forward refs to child components or DOM elements, enabling direct access to the DOM from parent components